utils/utils.py

```diff
@@ -112,15 +112,8 @@ def setup_ddp():
     master_addr = "127.0.0.1"
     master_port = "8889"
 
-    if os.getenv("LSB_DJOB_HOSTFILE") is not None:
-        ## source: https://www.olcf.ornl.gov/wp-content/uploads/2019/12/Scaling-DL-on-Summit.pdf
-        ## The following is Summit specific
```

Comment:
Let's keep this since I'll forget what LSB is

Reply:
I put back the comments.